spec/20-http_header_format.md

```diff
@@ -99,11 +99,11 @@ trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently only one bit is used. Se
 
 #### Trace-id
 
-Is the ID of the whole trace forest. It is represented as a 16-bytes array, for
```

Comment:
By the way, what is a *trace forest* supposed to be?
> ```#### Trace-id```
```Is the ID of the whole trace forest.```

Reply:
it is multi-tree concept. We will bring description back https://github.com/w3c/trace-context/issues/279